### ISSUE TITLE
Feature proposal: `meltano config validate`

### ISSUE THREAD

**Issue by aaronsteers** [meltano/meltano#7459]
This could also be invoked inline as part of `meltano test ...`.

We can also print inline as warnings:

1. Whenever a plugin is invoked.
2. After `meltano config set --interactive`.

Ideally these would also print requirements or explain requirements to users: `meltano config list`, and `meltano config set --interactive` prior to asking for input.

**Comment by DanielPDWalker** [comment]
My thoughts on this are also related to https://github.com/meltano/sdk/issues/1519, an issue about being able to know if an SDK plugin has valid settings based on user defined groups of settings.

If these groups of settings are in the SDK, and present in an SDK plugin when a user runs `meltano config set --interactive`, then one of the first things presented to the user could be these groups, and to make the user choose what group of settings they want to provide.

If the plugin doesn't have these groups, the behaviour stay the same where `--interactive` can loop through all of the settings. Or we could use the `settings group validation` to try to let the user choose a valid set of settings to provide.

**Comment by tayloramurphy** [comment]
@aaronsteers I think is a duplicate of a duplicate:

* https://github.com/meltano/meltano/issues/7285 which was used in favor of
* https://github.com/meltano/meltano/issues/3010

I like either validate / lint - though lint to me implies it's going to take action potentially on your file.

**Comment by aaronsteers** [comment]
This would be a duplicated of

- #3010 (now closed)

But not a dupe of:

- #7285 

Since #7285 needs to do much more than validating, its scope will necessarily be a superset of many component commands. We need to first write the component tests and actions before we can pair them into a wholistic 'lint' operation.

For example, #7285 mentions calling `test` on all plugins, but it doesn't itself redundant with the already-existing `test` command and the yet-to-be-built test capability for targets (#6855). This config validate command would probably be called in many different places, but the dev work is distinct.

**Comment by WillDaSilva** [comment]
Still relevant
